chpl/chpl-service/src/main/java/gov/healthit/chpl/report/criteriaattribute/TestToolReportDao.java

```diff
@@ -73,13 +75,15 @@ public List<TestToolReport> getTestToolReports() {
     }
 
     public List<TestToolListingReport> getTestToolListingReports() {
-        String hql = "SELECT cc.id as certificationCriterionId, tt.id as testToolId, cpd.chplProductNumber "
+        String hql = "SELECT cc, tt, cpd.chplProductNumber "
                 + "FROM CertificationCriterionEntity cc, "
+                + "CertificationEditionEntity ce, "
```

Comment:
Same question about excluding criteria that have no edition. I double checked and there is only 1 now but I imagine there would be more in the future.